addons/base_import_module/models/ir_module.py

```diff
@@ -258,6 +268,125 @@ def module_uninstall(self):
             modules_to_delete.unlink()
         return res
 
+    @api.model
+    def web_search_read(self, domain, specification, offset=0, limit=None, order=None, count_limit=None):
+        res = super().web_search_read(domain, specification, offset=offset, limit=limit, order=order, count_limit=count_limit)
+        if any(dom for dom in domain if dom[0] == 'module_type' and dom[-1] != 'official'):
```

Comment:
is the filter "All" supposed to also return industries? because it has an empty domain `[]` so skip this condition

Reply:
No, as we don't want to do RPC calls when we display All apps. And we need the 'module_type', to have categories in the side panel.